src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js

```diff
@@ -20,6 +20,20 @@ const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
   save () {
     return loc('password.expired.submit', 'login');
   },
+
+  showMessages () {
+    // render messages as text
+    const messagesObjs = this.options.appState.get('messages');
+    if (messagesObjs && messagesObjs.value.length) {
+      let content = messagesObjs.value.map((messagesObj) => {
+        return messagesObj.message;
+      });
+      content = this.options.settings.get('brandName')?
+        [loc('password.expiring.subtitle.specific', 'login', [this.options.settings.get('brandName')])]:
+        content;
+      this.add(`<div class="ion-messages-container">${content.join(' ')}</div>`, '.o-form-error-container');
+    }
+  },
```

Comment:
Hi @nikhilvenkatraman-okta , looks like I may have missed this while working on the ReEnroll view.
- Shouldn't this be in a `subtitle()` function or something like that?
- Trying to understand how generic this logic is? Like should we put it in `EnrollAuthenticatorPasswordView` instead?
- Also the message seems to be something else here under Password will expire soon section -https://oktawiki.atlassian.net/wiki/spaces/UX/pages/737092192/Sign+In+Widget+Design+-+IdX#Password 
I can't remember if there was a decision made over slack, anyhow, I don't like the message in the mock as it's too detailed, that's just my opinion though.

Reply:
would rather 
- either 
```
subtitle() { 
  if brandName then return loc('password.expiring.subtitle.specific', 'login', ...);
  else return ....
},
showMessages() {
  return null;
}
```
- or

```
subtitle() { 
  if brandName then return loc('password.expiring.subtitle.specific', 'login', ...);
},
showMessages() {
  if brandName then return null;
}
```

Reply:
@anipendakur-okta @haishengwu-okta I saw that on this screen the message ` When your password expires you will be locked out of your Okta account.` is coming from the server within the `messages`object which was being rendered by the `showMessages` function we have in `BaseForm`.  I was keeping it consistent with what we did in other places by overriding the `showMessages` at the view layer and add the customization logic.

I can render this as a subtitle  but that would mean we would extract `messages` object within the `subtitle` function and return null from the `showMessages ` function, just so that we can render this as a subtitle. Visually there is no difference. In all other places we render `messages` inside the ion-message-container but here we would render as a subtitle.

I am curious why this one-off is better  ?

Reply:
when we have a `brandName` we are just displaying a static text with the brandName as a parameter. In that case a subtitle makes sense. When we don't have a brandName we can override the showMessages and render text within the ion-messages-container. This would need us to check for brandName
```
  subtitle () {
    if (this.options.settings.get('brandName')) {
      return loc('password.expiring.subtitle.specific', 'login', [this.options.settings.get('brandName')]);
    }
  },

  showMessages () {
    if (this.options.settings.get('brandName')) {
      return null;
    }
    // render messages as text
    const messagesObjs = this.options.appState.get('messages');
    if (messagesObjs && messagesObjs.value.length) {
      let content = messagesObjs.value.map((messagesObj) => {
        return messagesObj.message;
      });
      this.add(`<div class="ion-messages-container">${content.join(' ')}</div>`, '.o-form-error-container');
    }
  },

```
I prefer this as this seems to be clear and in a way consistent with what we are doing in other places. Let me know what you guys think.

Reply:
I guess you would do in `initialize` function
```
initialize() {
  super.initialize();
  if brand name exists {
    this.subtitle = () => { return loc(...); }
    this.showMessages = () => {}; // literally no-op
  }
}
```

Reply:
I think I like @nikhilvenkatraman-okta 's solution better for a few reasons. 
* We are using functions that probably stick to `prototype` or whatever `View.extend({})` does, that way we're following courage guidelines.
* `this.subtitle = () => {return loc();}` will force the function on the instance level which seems a bit hacky to me.

Reply:
the problem is `showMessages` has duplicated code form its parent unless we can do
```
showMessages() {
  if brand name then return null
  else super.showMessages();
}
```

Reply:
updated to call `showMessages ` from BaseForm